test/plugins/features/movies/controller.test.js

```diff
@@ -23,5 +23,128 @@ describe('movie controller', () => {
     });
 
   });
-  
+
+
+  describe('retrieve a list of movies', () => {
+
+    it('returns all movies', () => {
+      return new Movie().fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((count) => {
+        return [Controller.display(), count];
+      })
+      .spread((movies, count) => {
+        expect(movies).to.exist;
+        expect(movies).to.have.length(count);
+      });
+    });
+
+  });
+
+  describe('filters movies based on specified dates', () => {
+
+    it('returns movies in 2000', () => {
+      return new Movie().query(function(qb) {
+        qb.where('release_year', '=', '2000');
```

Comment:
optional nit: Can also be `qb.where('release_year', 2000);`. Either way, since the field is a number in the database, I'd keep it as `2000` instead of `'2000'`.